### PR TITLE
feat: Update Helpers.java

### PR DESCRIPTION
**Pull request by priyanshul-govil**
Details in individual commits.

### CODE REVIEW

**Comment by priyanshul-govil** [comment]
Need code review on each commit. When all commits are made for Helpers.java, I'll merge on my own.

**Review by viveknathani — COMMENTED** [review]
looks good to me!

**Review by viveknathani — APPROVED** [review]
approved

**Review by viveknathani — APPROVED** [review]
perfect

File: src/secur3dit/crypto/Helpers.java [hunk 1]
```diff
@@ -92,4 +92,129 @@ public static void addRoundKey(char[] state, final char[] key) {
...
+     * @param K     The initial key (char[32]) to be expanded into round keys.
+     * @return W    The final 15 round keys.
+     */
+    public static char[][] keyExpansion(char[] K) {
+
+        /**
+         * This inner class contains helper functions for keyExpansion.
+         * Details at: https://en.wikipedia.org/wiki/AES_key_schedule#The_key_schedule
+         */
+        final class keyExpansionHelpers {
```
Comment (viveknathani) [inline src/secur3dit/crypto/Helpers.java]: class names should be `UpperCamelCase`

Comment (priyanshul-govil) [reply src/secur3dit/crypto/Helpers.java]: done

**Comment by priyanshul-govil** [comment]
Self-merging this PR.

**Pull request merged**